lib/omnibus/fetchers/git_fetcher.rb

```diff
@@ -165,7 +171,9 @@ def git_checkout
     def git_fetch
       fetch_cmd = "fetch #{source_url} #{described_version}"
       fetch_cmd << " --recurse-submodules=on-demand" if clone_submodules?
-      git(fetch_cmd)
+      retry_block("git fetch", [CommandTimeout, CommandFailed]) do
```

Comment:
It might be nice to know the full fetch command that times out...maybe we can do this:

```ruby
retry_block(fetch_cmd, [CommandTimeout, CommandFailed]) do
```